WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -594,6 +615,77 @@ private extension ProductFormViewController {
     }
 }
 
+// MARK: - Tooltip
+//
+private extension ProductFormViewController {
+    func tooltipTargetPoint() -> CGPoint {
+        guard let indexPath = findDescriptionAICellIndexPath() else {
+            return .zero
+        }
+
+        guard let cell = tableView.cellForRow(at: indexPath),
+              let buttonCell = cell as? ButtonTableViewCell,
+              let imageView = buttonCell.button.imageView else {
+            return .zero
+        }
+
+        let rectOfButtonInTableView = tableView.convert(buttonCell.button.frame, from: buttonCell)
+
+        return CGPoint(
+            x: rectOfButtonInTableView.minX + imageView.frame.midX,
+            y: rectOfButtonInTableView.maxY
+        )
+    }
+
+    func configureTooltipPresenter() {
+        guard tooltipUseCase.shouldShowTooltip && tooltipPresenter == nil else {
```

Comment:
We can add the guard check for `aiEligibilityChecker.isFeatureEnabled(.description)` here at the beginning of the method.

Reply:
Added as part of 8fa8fe32e661f4d32d26f8ef1f19d90111cd3b09